modules/streamer/src/main/YouTubeApi.scala

```diff
@@ -20,7 +20,7 @@ final private class YouTubeApi(
     net: NetConfig
 )(using Executor, akka.stream.Materializer):
 
-  private var lastResults: List[YouTube.Stream] = List()
+  private var lastResults: List[YouTube.Stream] = Nil
```

Comment:
I could be persuaded by `List.empty[YouTube.stream]` here (omitting the LHS ascription).